pheps/phep-0001.md

```diff
@@ -0,0 +1,287 @@
+PHEP: 1
+Title: PHEP Purpose and Guidelines
+Author: Jonathan Niehof <jtniehof@gmail.com>
+Discussions-To: https://github.com/heliophysicsPy/standards/pull/22
+Status: Draft
+Type: Process
+Content-Type: text/markdown; charset=UTF-8; variant=CommonMark
+Created: 20-Oct-2023
+Post-History: 24-Oct-2023
+
+# What is a PHEP?
+PHEP stands for PyHC Enhancement Proposal. A PHEP is a design document providing information to the Python in Heliophysics Community (PyHC), describing its processes, or describing a new standard for its packages. The PHEP should provide a concise technical specification of the feature or standard and a rationale.
+
+We intend PHEPs to be the primary mechanisms for proposing major new standards, for collecting community input on an issue, and for documenting the design decisions that have gone into PyHC standards and recommendations. The purpose of the PHEP process is to develop and document consensus within the community. The PHEP author is responsible for building this consensus and documenting dissenting opinions.
+
+Because the PHEPs are maintained as text files in a versioned repository, their revision history is the historical record of the feature proposal. This historical record is available by the normal git commands for retrieving older revisions, and can also be browsed on GitHub.
+
+# PHEP Motivation
+The [PyHC standards](https://doi.org/10.5281/zenodo.2529131) have not been updated since their initial approval in 2018, and indeed there is no established process for updating them. PyHC discussions often involve what packages "should" do, or "should be required" to do. For standards to be effective, they need to be seen as legitimate by those they may be imposed on.
+
+A mechanism is needed to capture the sense of the community and allow it to progress while recognizing that the community is broad, diffuse, and diverse, and not all stakeholders are able to be present for all conversations.
+
+# PHEP Audience
+<a name="phep-audience"></a>
+The typical primary audience for PHEPs are the developers of PyHC packages.
```

Comment:
```suggestion
The typical primary audience for PHEPs are the developer and user communities for PyHC packages.
```
I'm not sure if many users would be interested in participating in these discussions, but their thoughts, ideas, and questions would be really helpful!

Reply:
User feedback is always good!